BedrockServer.cpp

```diff
@@ -134,6 +136,11 @@ void BedrockServer::sync(SData& args,
     // If still no value, use the number of cores on the machine, if available.
     workerThreads = workerThreads ? workerThreads : max(1u, thread::hardware_concurrency());
 
+    // A minumum of *2* worker threads are required. One for blocking writes, one for other commands.
+    if (workerThreads < 2) { 
```

Comment:
Why not Justin change the max call above to be 2u?

Reply:
Because if someone specified `-workerThreads 1` they would still get 1 in that case.

Reply:
Oh duh